### PR TITLE
Add history for future release

### PR DESCRIPTION
**Pull request by ablwr**
Here is a summary of the work since the past release.

@dericed @JeromeMartinez Let me know if this should be release 1.3.9, 1.4, or 2.0.

Er, and maybe whitespace trimming?

### CODE REVIEW

**Comment by ablwr** [comment]
Whoops -- or version 20.05 :)

**Comment by JeromeMartinez** [comment]
`1.4` is fine for me, but `20.05` may be also a good option :).

File: History_CLI.txt [hunk 1]
```diff
@@ -4,26 +4,32 @@ Notes:
...
-
-Version 1.3.5, 2018-08-27
--------------
-x Fix missing Null Character on LIST-INFO chunks during file writing
-
-Version 1.3.3, 2017-11-21
--------------
-x Fix regression preventing to store bext chunck version 0 or 1
-
+Version 1.4.0, 2020-05-20
```
Comment (JeromeMartinez) [inline History_CLI.txt:7]: You forgot to update this line ;-).
(version + date + line about the version numbering)

File: History_GUI.txt [hunk 2]
```diff
@@ -4,28 +4,50 @@ Notes:
...
+x Fix duplication of CodingHistory headers (Issue #103)
+x Stop setting BextVersion field twice (Issue #98)
+x Fix occasional crash when saving large number of files (Issue #71)
+x Improve file drag and drop events (Issue #83)
+x Improve CodingHistory dialog (Issue #122, #124)
++ Add icon to close files (Issue #100)
++ Show available info for invalid files (Issue #93)
++ Add warnings and use icons for errors/warnings (Issue #101)
++ Soften color palette (#126)
+x Transition to year.month-based version numbering
```
Comment (JeromeMartinez) [inline History_GUI.txt:27]: I prefer a "+" here, "x" is more for bug fixes.

**Comment by dericed** [comment]
:)

**Pull request merged**